docs/modules/ROOT/pages/ios_security.adoc

```diff
@@ -0,0 +1,163 @@
+You will then see the certificate’s details, starting with the validation status. 
+
+=== More Information
+
+For more information, please refer to {security-information-url}[the security information].
+
+== Data
+
+=== Separation
+
+Separate directories are used for the data of every server connection.
+This provides the following benefits:
+
+- A strong barrier against accidentally spilling data between different connections.
+- All data relating to a connection can be deleted by deleting the respective directory.
+
+=== Encryption
+
+The app uses the {ios-filesystem-encryption-url}[filesystem encryption built into iOS]. 
+Using the {apple-completeuntilfirstuserauthentication-url}[`CompleteUntilFirstUserAuthentication`] file protection, data can't be accessed after a restart until the device has been unlocked once by the user.
+
+=== Sync
+
+The {ios-sdk-sync-strategies-url}[Sync Strategies], planned to be used in the app, focus on preventing data loss locally and remotely.
+
+=== Secure Document View
+
+HTML and Microsoft Office document content is viewed using {apple-wkwebview-url}[`WKWebView`], which renders the content in a separate process. 
+Additional hardening is achieved by disabling JavaScript and blocking all network requests, which protects against lesser known, non-obvious attacks like {css-keylogging-url}[CSS Keylogging].
+
+=== Passcode
+
+Users can set a Passcode to control access to the app.
+Find out more about this in xref:ios_settings.adoc#passcode[the Passcode section of the Settings documentation].
+
+== Miscellaneous
+
+=== Continuous Integration (CI)
+
+Continuous Integration tests verify that central security mechanisms and assumptions work as expected, covering areas such as _redirections_, _certificate handling_, common Man-in-the-middle (MITM) attack scenarios, and the secure storage of authentication secrets.
+
+=== SQL Injection
+
+To protect against SQL injection attacks, parameters are never made part of the SQL statements themselves. 
+Instead, placeholders are used and the parameters are subsequently bound to the SQL statements.
+For example, instead of running a query, such as `SELECT * FROM users WHERE name='John Doe'`, the query would be parameterised, such as: `SELECT * FROM users WHERE name=:nameToSearchFor`.
+
+=== Reproducibility
+
+The build script that created the {ios-sdk-openssl-lib-url}[OpenSSL binaries] used in the app is available in the SDK's {ios-sdk-openssl-build-script-url}[GitHub repository] and can be used to reproduce the build result.
+
+NOTE: OpenSSL is used solely to provide detailed summaries of SSL/TLS certificates - functionality that iOS is currently missing.
+
+=== Logging
+
+When logging information, parts of the log message can be tagged as private. 
+If "*Mask private data*" is enabled, under menu:Settings[Logging] (it is by default), these parts will be replaced with `«private»` before the log message is written.
+
+image:masking-private-data.png[]
```

Comment:
`Mask private Data` was removed from the app. This section can be deleted.

Reply:
Solved with 9e221ab6d78890997cb59a5a6f57f2f4e43ccb0b